site/tutorials/keptn-notification-service.md

```diff
@@ -0,0 +1,277 @@
+summary: Create a Keptn Notification Service
+id: keptn-notification-service
+categories: tutorial
```

Comment:
can you please change it to `keptn,integration`

Reply:
`categories` or `tags`?

Reply:
categories